core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1725,6 +1725,17 @@ class SparkContext(config: SparkConf) extends Logging {
     }
   }
 
+
+  private[spark] def decommissionExecutors(executorIds: Seq[String]): Unit = {
```

Comment:
Do you plan to make this a public API in the future? I don't see the usage of it in this PR.

Reply:
Good catch, I'll drop this.